include/adt/edgeset.h

```diff
@@ -78,8 +80,8 @@ edge_set_remove(struct edge_set **set, unsigned char symbol);
 void
 edge_set_remove_state(struct edge_set **set, fsm_state_t state);
 
-void
-edge_set_compact(struct edge_set **set,
+int
```

Comment:
This and `edge_set_replace_state` can now fail, but only due to allocation failure.